app/views/standup_meeting_groups/index.html.erb

```diff
@@ -1,13 +1,19 @@
-<h1>Standup Meeting Groups</h1>
-
 <% if policy(:standup_meeting_group).new? %>
   <div class="p-2 border-black border-2" id="new_standup_meeting_group">
     <%= render 'form', standup_meeting_group: StandupMeetingGroup.new %>
   </div>
 <% end %>
 
-<hr class="mb-5 mt-5 border-b-2">
+<div class="flex flex-col">
+  <h1 class="font-bold text-3xl">My Standups</h1>
+  <div id="my-standup-meeting-groups" class="grid grid-cols-3 gap-3 pl-5 pt-5 pb-5">
+    <%= render StandupMeetingGroup::JoinedCardComponent.with_collection(@my_standup_meeting_groups, current_user:) %>
+  </div>
+</div>
 
-<ul role="list" id="standup_meeting_group_list">
-  <%= render partial: 'standup_meeting_group_list_item', collection: @standup_meeting_groups, as: :standup_meeting_group %>
-</ul>
\ No newline at end of file
+<div class="flex flex-col">
+  <h1 class="font-bold text-3xl">Find a Standup</h1>
```

Comment:
Again notice how this heading is centered in the mobile wireframes. This also probably shouldn't be an `<h1>` (only one should be on the page).